config/dotenv.js

```diff
@@ -0,0 +1,8 @@
+module.exports = function(/* env */) {
+    return {
+      clientAllowedKeys: [],
+      // Fail build when there is missing any of clientAllowedKeys environment variables.
+      // By default false.
+      failOnMissingKey: false
```

Comment:
I was hoping that we could show an empty value in `.env-sample` (equivalently, `.env`) so that `failOnMissingKey: true` will result in a build error if a developer tried to run `ember serve` without passing a token.

```
# .env

GITHUB_API_KEY=
```

```javascript
// config/dotenv.js

module.exports = function(env) {
  return {
    clientAllowedKeys: ['GITHUB_API_KEY'],
    // Fail build when there is missing any of clientAllowedKeys environment variables.
    // By default false.
    failOnMissingKey: true
  };
};
```

I tried this and found out the build succeeds (of course, we're unable to fetch data from GitHub). It seems, the only way to get the build to error is if `.env` is completely missing `GITHUB_API_KEY`. This wouldn't make a good solution so let's keep the `.env-sample` that you made.